primeqa/qg/README.md

```diff
@@ -1,12 +1,73 @@
-# Question Generation for TableQA
-Generates synthetic question-answer pairs for tables. We first sample SQL queries from a given table, and then use a text-to-text transformer (T5) to transcribe the SQL query to a natural language question. For more details on the method check out our EMNLP 2021 paper [here](https://arxiv.org/abs/2109.07377).
-Currently we support tableqg model training over wikisql data and passageqg model training over squad, squad_v2, tydiqa data. 
+# Question Generation (QG)
+The task of QG can be summarized as given a table pr passage, generate <question, answer> pairs such that the answer for that question can be deduced from the table/passage.  
+The primary script for QG is [run_qg.py](./run_qg.py).  This runs a transformer-based sequence generation pipeline as shown below (for TableQG):
+## Pipeline
+<img src="../../docs/img/tableqg_pipeline.png" width="500" class="center">
 
-To use our pre-trained QG models with only few lines of code check out the inference notebooks here: ../notebooks/qg/tableqg_inference.ipynb, ./notebooks/qg/pasageqg_inference.ipynb. 
+The above pipeline shows the pipeline for generating questions from a table. PasageQG follows a similar pipleine, except the SQL generator over a table is replaced by the Answer Sampler over a passage which tries to sample answer token(s) from the given passage and then generate questions having those answer(s). 
```

Comment:
I think some references to the appropriate files would be helpful here

Reply:
Done.